src/i_video.c

```diff
@@ -1698,6 +1699,67 @@ void I_InitGraphics(void)
     ResetLogicalSize();
 }
 
+static struct {
+    int w, h;
+} native_res[] = {
+    {  320,  240 },
+    {  640,  480 },
+    {  800,  600 },
+    { 1024,  768 },
+    { 1280, 1024 },
+    { 1280,  720 },
+    { 1280,  800 },
+    { 1366,  768 },
+    { 1440,  900 },
+    { 1440, 1080 },
+    { 1680, 1050 },
+    { 1920, 1080 },
+    { 1920, 1200 },
+    { 2048, 1152 },
+    { 2560, 1080 },
+    { 2304, 1440 },
+    { 2560, 1600 },
+    { 3200, 2400 },
+    { 3440, 1440 },
+    { 3840, 2160 },
+    { 5120, 2160 }  
+};
```

Comment:
There's also 3840x1600 (aspect ratio of 2.4 that will be our max for next release).